cmd/nerdctl/run_network.go

```diff
@@ -164,6 +167,48 @@ func generateNetOpts(cmd *cobra.Command, dataStore, stateDir, ns, id string) ([]
 				ports = append(ports, pm...)
 			}
 		}
+	case nettype.Container:
+		if runtime.GOOS != "linux" {
```

Comment:
I'm not sure whether we need to support this feature in rootless mode or not? cc @AkihiroSuda

Reply:
Why not?